eval.go

```diff
@@ -104,7 +103,7 @@ func (d *indexData) Search(ctx context.Context, q query.Q, opts *SearchOptions)
 	default:
 	}
 
-	tr := trace.New("indexData.Search", d.file.Name())
+	tr, ctx := trace.New(ctx, "indexData.Search", d.file.Name())
```

Comment:
The `github.com/google/zoekt/trace` package replaces `golang.org/x/net/trace` and presents the same API. This is the same pattern as in the main repository ([example](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/cmd/frontend/graphqlbackend/search.go?subtree=true#L74:17))